api/opportunities/models/opportunities.settings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "kind": "collectionType",
+  "collectionName": "opportunities",
+  "info": {
+    "name": "Opportunities",
+    "description": ""
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true
+    },
+    "skills": {
+      "collection": "interest"
+    },
+    "level": {
+      "type": "enumeration",
+      "enum": [
+        "Beginner",
+        "Intermediate",
+        "Advanced"
+      ],
+      "required": true
+    },
+    "commitmentHoursPerWeek": {
+      "type": "integer",
+      "required": true
+    },
+    "description": {
+      "type": "richtext",
+      "required": true
+    },
+    "expectations": {
+      "type": "component",
+      "repeatable": true,
+      "component": "positions.expectations",
```

Comment:
What's the difference between positions and opportunities?

Reply:
There's no difference, I just reused the component.